roles/ostree_admin_unlock/tasks/main.yml

```diff
@@ -7,7 +7,8 @@
 - name: Unlock system
   command: ostree admin unlock
 
-- include: '../../rpm_ostree_status/tasks/main.yml'
+- include_role:
```

Comment:
Let's only use import_role as a baseline and only use include_role where we need it dynamically called.  I ran into a recursion depth limit using include.